src/Nethermind/Nethermind.Consensus/Validators/TxValidator.cs

```diff
@@ -161,15 +161,19 @@ public ValidationResult IsWellFormed(Transaction transaction, IReleaseSpec relea
             { To: null } => TxErrorMessages.TxMissingTo,
             { MaxFeePerBlobGas: null } => TxErrorMessages.BlobTxMissingMaxFeePerBlobGas,
             { BlobVersionedHashes: null } => TxErrorMessages.BlobTxMissingBlobVersionedHashes,
-            _ => ValidateBlobFields(transaction)
+            _ => ValidateBlobFields(transaction, releaseSpec)
         };
 
-    private ValidationResult ValidateBlobFields(Transaction transaction)
+    private ValidationResult ValidateBlobFields(Transaction transaction, IReleaseSpec releaseSpec)
     {
         int blobCount = transaction.BlobVersionedHashes!.Length;
-        ulong totalDataGas = BlobGasCalculator.CalculateBlobGas(blobCount);
-        return totalDataGas > Eip4844Constants.MaxBlobGasPerTransaction ? TxErrorMessages.BlobTxGasLimitExceeded
-            : blobCount < Eip4844Constants.MinBlobsPerTransaction ? TxErrorMessages.BlobTxMissingBlobs
+        if (!releaseSpec.IsEip7742Enabled
```

Comment:
same issue?

Reply:
same thing here, need to skip the logic related to `MaxBlobGasPerBlock` which is equal to `MaxBlobGasPerTransaction`